test/functional/test_framework/test_node.py

```diff
@@ -34,7 +34,11 @@ def __init__(self, i, dirname, extra_args, rpchost, timewait, binary, stderr, mo
         self.index = i
         self.datadir = os.path.join(dirname, "node" + str(i))
         self.rpchost = rpchost
-        self.rpc_timeout = timewait
+        if timewait:
+            self.rpc_timeout = timewait
+        else:
+            # Wait for up to 60 seconds for the RPC server to respond
+            self.rpc_timeout = 60
```

Comment:
Previously we used a default of `HTTP_TIMEOUT = 30`.

Why was this changed?

Reply:
Longer is better (I think?). Travis is so unpredictable that one minute seemed appropriate.

We can tune it back down if you think that's better.